msal/src/main/java/com/microsoft/identity/client/claims/ClaimsRequest.java

```diff
@@ -46,6 +46,28 @@
     private List<RequestedClaim> mAccessTokenClaimsRequested = new ArrayList<>();
     private List<RequestedClaim> mIdTokenClaimsRequested = new ArrayList<>();
 
+    @Override
```

Comment:
nit : `CheckStyle` and `PMD` will fail for these generated methods. Add `@SuppressWarnings("PMD")` and 

`//CHECKSTYLE:OFF` above the method and `//CHECKSTYLE:ON` at the end of the method